heroku.yml

```diff
@@ -14,4 +14,4 @@ release:
   image: web
 # The command that runs your application. Replace 'app' with the name of your app.
 run:
-  web: gunicorn --max-requests 1000 --max-requests-jitter 50 -t 180 --log-level debug councilmatic.wsgi:application
+  web: gunicorn --max-requests 1000 --max-requests-jitter 50 -t 20 --log-level debug councilmatic.wsgi:application
```

Comment:
I chose to start with 20 seconds because we've got some slower database requests, especially for search queries. But we can also try lower.